### PR TITLE
implement APM-120

### PR DESCRIPTION
**Pull request by jsteemann**
### Scope & Purpose

APM-120: https://arangodb.atlassian.net/browse/APM-120
Docs PR: https://github.com/arangodb/docs/pull/753

Add pseudo log topic "all" to set the log levels for all log topics at once. For example, this can be used when starting a server with trace or debug logging enabled for all log topics, e.g. `--log.level all=debug` or `--log.level all=trace`.

This is very coarse and should only be used for such use cases.

- [ ] :hankey: Bugfix (requires CHANGELOG entry)
- [x] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification
- [x] :book: CHANGELOG entry made

#### Backports:

- [x] No backports required

#### Related Information

- [x] Docs PR: https://github.com/arangodb/docs/pull/753

### Testing & Verification

- [x] The behavior in this PR was *manually tested*
  - [x] Added new **integration tests** (e.g. in shell_client)

### CODE REVIEW

File: arangod/RestHandler/RestAdminLogHandler.cpp [hunk 1]
```diff
@@ -425,8 +425,15 @@ void RestAdminLogHandler::handleLogLevel() {
     if (slice.isString()) {
       Logger::setLogLevel(slice.copyString());
     } else if (slice.isObject()) {
+      if (VPackSlice all = slice.get("all"); all.isString()) {
```
Comment (gnusi) [inline arangod/RestHandler/RestAdminLogHandler.cpp:428]: can we have a constant for that?

Comment (cpjulia) [reply arangod/RestHandler/RestAdminLogHandler.cpp:428]: Yes perhaps`VPackSlice const all` as doesn't have its value changed somewhere else?

**Review by cpjulia — APPROVED** [review]
LGTM

File: arangod/RestHandler/RestAdminLogHandler.cpp (hunk 1 above)
Comment (gnusi) [reply arangod/RestHandler/RestAdminLogHandler.cpp:428]: No, I meant a constant for "all" topic name

**Pull request merged**